`Kiss.hs` fails to compile (duplicate field label `kCels`)

Do you want to request a feature or report a bug?

Bug.

What is the current behavior?

`Kiss.hs` fails to compile on the master branch (436c673b29afa7212a692b8cc40ed4e4637457ae):

```haskell
/home/huggablemonad/projects/smooch/app/src/Kiss.hs:21:28: error:
    Multiple declarations of ‘kCels’
    Declared at: src/Kiss.hs:9:26
                 src/Kiss.hs:21:28
```

What is the expected behavior?

There shouldn't be any compilation errors.

Which browser / OS are affected by this issue?

All.

--- a/app/src/Kiss.hs
+++ b/app/src/Kiss.hs
@@ -6,7 +6,7 @@ import           Data.Aeson hiding (Array)
 import           Data.Array (Array)
 
 data KissSet = KissSet { kData    :: CNFKissData
-                       , kCels   :: [CNFKissCel]
+                       , kCnfCels :: [CNFKissCel]
                        , kPalette :: Array Int PaletteFilename
                        } deriving (Eq, Show)
 

